addons/html_builder/__manifest__.py

```diff
@@ -21,6 +21,10 @@
     'depends': ['base', 'html_editor', 'mail'],
 
     'assets': {
+        'web._assets_primary_variables': [
```

Comment:
not sure about this.  i feel like it should be somehow in the html_builder.assets bundle